sdk/storage/azure-storage-blob/azure/storage/blob/_deserialize.py

```diff
@@ -47,3 +50,47 @@ def get_page_ranges_result(ranges):
     if ranges.clear_range:
         clear_range = [{'start': b.start, 'end': b.end} for b in ranges.clear_range]
     return page_range, clear_range  # type: ignore
+
+
+def get_modify_conditions(kwargs):
+    # type: (Dict[str, Any]) -> ModifiedAccessConditions
+    if_match = None
+    if_none_match = None
+    match_condition = kwargs.pop('match_condition', None)
+    if match_condition == MatchConditions.IfNotModified:
+        if_match = kwargs.pop('etag', None)
+    elif match_condition == MatchConditions.IfPresent:
+        if_match = '*'
+    elif match_condition == MatchConditions.IfModified:
+        if_none_match = kwargs.pop('etag', None)
+    elif match_condition == MatchConditions.IfMissing:
+        if_none_match = '*'
+
+    return ModifiedAccessConditions(
+        if_modified_since=kwargs.pop('if_modified_since', None),
+        if_unmodified_since=kwargs.pop('if_unmodified_since', None),
+        if_match=if_match or kwargs.pop('if_match', None),
+        if_none_match=if_none_match or kwargs.pop('if_none_match', None)
+    )
+
+
+def get_source_conditions(kwargs):
```

Comment:
Could we please create a _serialize.py and put these methods there?

Reply:
Done :)